doc/release-notes.md

```diff
@@ -63,6 +63,12 @@ P2P and network changes
 
 Updated RPCs
 ------------
+- The top-level fee fields `fee`, `modifiedfee`, `ancestorfees` and `descendantfees`
+  returned by RPCs `getmempoolentry`,`getrawmempool(verbose=true)`,
+  `getmempoolancestors(verbose=true)` and `getmempooldescendants(verbose=true)`
+  are now deprecated and will be removed in the next major version (use
+  `-deprecated=fees` if needed in this version). The same values, denominated
+  in BTC, can be accessed through the `fees` object in the result. (#22689)
```

Comment:
The units don't seem to be changed and remain in BTC/kvB?

Feel free to ignore (the help prints the wrong unit as well, I believe)
```suggestion
  in BTC/kvB, can be accessed through the `fees` object in the result. (#22689)
```

Reply:
yeah, i think the help is incorrect here. i can open a general RPC cleanup PR. for starters:

- address where argument order doesn't match help order
- fix fee vs feerate and other small errors

Reply:
So, `fee` and `modifiedfee` are uniformly in BTC/kvB. OTOH, it looks to me like `ancestorfees` and `descendantfees` are in sat/vB in the deprecated fields and in BTC/kvB in the `fees` object (urgh, we've been moving from BTC/kvB to sat/vB elsewhere).

If we mention this in the release notes (and I think it's a good idea as @glozow suggested), it may be best to (a) use the correct units of BTC/kvB and (b) clarify which fields change from sat/vB to BTC/kvB (`ancestorfees` and `descendantfees`) in the release notes.

Finally, here's a commit in https://github.com/jonatack/bitcoin/commits/pr22689-help-fixups that both fixes and improves the docs for these fields, feel free to use or pull it in here since these lines are already being touched (or later, if you prefer) as the helps could use some...help :)

Reply:
Note that the units for ancestor and descendant fees are different - they were in satoshis and are now in BTC